src/dbus/NhekoDBusApi.h

```diff
@@ -58,6 +58,26 @@ class RoomInfoItem : public QObject
     int unreadNotifications_;
 };
 
+//! Get the nheko D-Bus API version.
+QString
+apiVersion();
+//! Get the nheko version.
+QString
+nhekoVersionString();
```

Comment:
version vs versionString?

Reply:
True. I'd done that when it was QString vs. QVersionNumber.